packages/web-react/src/components/Tabs/useTabsStyleProps.ts

```diff
@@ -1,6 +1,9 @@
 import classNames from 'classnames';
+import { CSSProperties } from 'react';
 import { useClassNamePrefix } from '../../hooks';
+import { useSpacingStyle } from '../../hooks/useSpacingStyle';
```

Comment:
again, can it be imported from the same file as useClassNamePrefix?
```suggestion
import { useClassNamePrefix, useSpacingStyle } from '../../hooks';
```